templates/shared/forms/interactive/managed-apps.html

```diff
@@ -18,124 +33,143 @@ <h3 class="p-heading--five">What apps are you using?</h3>
-            <input type="checkbox" id="where-running-Public-cloud-kubernetes">
-            <label for="where-running-Public-cloud-kubernetes">Kubernetes</label>
+            <h4 class="p-heading--five">App requirements</h4>
+            <input type="checkbox" id="requirements-apps-on-kubernetes">
+            <label for="requirements-apps-on-kubernetes">Apps on Kubernetes</label>
+            <input type="checkbox" id="requirements-high-availability">
+            <label for="requirements-high-availability">High availability</label>
+            <input type="checkbox" id="requirements-24-7-operations">
+            <label for="requirements-24-7-operations">24/7 operations</label>
+            <input type="checkbox" id="requirements-update-scheduling">
+            <label for="requirements-update-scheduling">Update scheduling</label>
+            <input type="checkbox" id="requirements-security-patching">
+            <label for="requirements-security-patching">Security patching</label>
+            <input type="checkbox" id="requirements-monitoring">
+            <label for="requirements-monitoring">Monitoring</label>
           </div>
           <div class="col-4 u-sv3">
-            <input type="checkbox" id="where-running-Public-cloud-openstack">
-            <label for="where-running-Public-cloud-openstack">OpenStack</label>
-            <input type="checkbox" id="where-running-Public-cloud-oracle">
-            <label for="where-running-Public-cloud-oracle">Oracle</label>
-            <input type="checkbox" id="where-running-Public-cloud-vmware">
-            <label for="where-running-Public-cloud-vmware">VMWare</label>
-            <input type="checkbox" id="where-running-Public-cloud-cisco">
-            <label for="where-running-Public-cloud-cisco">Cisco</label>
+            <h4 class="p-heading--five">Use-case</h4>
+            <input type="checkbox" id="use-case-ai-ml">
+            <label for="use-case-ai-ml">AI / ML</label>
+            <input type="checkbox" id="use-case-big-data">
+            <label for="use-case-big-data">Big data</label>
+            <input type="checkbox" id="use-case-analytics">
+            <label for="use-case-analytics">Analytics</label>
+            <input type="checkbox" id="use-case-hpc">
+            <label for="use-case-hpc">HPC</label>
+            <input type="checkbox" id="use-case-telco-nfv">
+            <label for="use-case-telco-nfv">Telco NFV</label>
+            <input type="checkbox" id="use-case-web-apps">
+            <label for="use-case-web-apps">Web apps</label>
+            <input type="checkbox" id="use-case-other">
+            <label for="use-case-other">Other</label>
           </div>
+        </div>
+      </div>
+      <div class="js-formfield">
+        <h3 class="p-heading--five">Are you already running the application?</h3>
+        <div class="row u-no-padding u-equal-height">
           <div class="col-4 u-sv3">
```

Comment:
I think this could be a col-12 too
```suggestion
          <div class="col-12 u-sv3">
```